components/encryption/src/metrics.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use prometheus::*;
+
+lazy_static::lazy_static! {
+    pub static ref ENCRYPTION_DATA_KEY_GAUGE: IntGauge = register_int_gauge!(
+        "tikv_encryption_data_key_storage_total",
+        "Total number of ecryption data keys in use"
+    ).unwrap();
+    pub static ref ENCRYPT_DECRPTION_FILE_HISTOGRAM: HistogramVec = register_histogram_vec!(
+        "tikv_encryption_write_read_file_duration_seconds",
+        "Histogram of writing or reading file duration",
+        &["operation"],
+        exponential_buckets(0.001, 2.0, 16).unwrap() // Up to 65.5 seconds
+    ).unwrap();
+    pub static ref FILE_SIZE_GAUGE: IntGaugeVec = register_int_gauge_vec!(
```

Comment:
change to a better name to at least suggest relation with encryption?

Reply:
Fixed